workflow/rules/commons/05_refcon.smk

```diff
@@ -17,7 +17,7 @@ import pandas
 
 localrules: refcon_run_dump_manifest, refcon_cache_manifests
 
-REFCON_FOLDER = pathlib.Path(config['reference_container_folder'])
+DIR_REFCON = pathlib.Path(config['reference_container_folder'])
```

Comment:
same here: if that option does not exist in the config, this would produce an error

Reply:
I hope I addressed that issue appropriately in the latest commit